src/nvim/testdir/test_cursor_func.vim

```diff
@@ -0,0 +1,54 @@
+" Tests for cursor().
```

Comment:
New tests must be added to `src/nvim/testdir/Makefile`. I will did that when merging.